manager/controllers/clusters/cves.go

```diff
@@ -0,0 +1,141 @@
+package clusters
+
+import (
+	"app/base/models"
+	"app/manager/base"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+	"gorm.io/gorm"
+)
+
+var getClusterCvesAllowedFilters = []string{base.SortQuery, base.LimitQuery, base.OffsetQuery,
+	base.SearchQuery, base.PublishedQuery, base.SeverityQuery, base.CvssScoreQuery,
+	base.AffectedImagesQuery}
+
+var getClusterCvesFilterArgs = map[string]interface{}{
+	base.SortFilterArgs: base.SortArgs{
+		SortableColumns: map[string]string{
+			"id":             "cve.id",
+			"cvss_score":     "COALESCE(cve.cvss3_score, cve.cvss2_score, 0.0)",
+			"severity":       "cve.severity",
+			"publish_date":   "cve.public_date",
+			"synopsis":       "cve.name",
+			"images_exposed": "images_exposed",
+		},
+		DefaultSortable: []base.SortItem{{Column: "id", Desc: false}},
+	},
+	base.SearchQuery: base.CveSearch,
+}
+
+// GetClusterCvesSelect
+// @Description CVE in cluster data
+// @Description presents in response
+type GetClusterCvesSelect struct {
+	Cvss2Score    *float32         `json:"cvss2_score"`
+	Cvss3Score    *float32         `json:"cvss3_score"`
+	Description   *string          `json:"description"`
+	Severity      *models.Severity `json:"severity"`
+	PublicDate    *time.Time       `json:"publish_date"`
+	Name          *string          `json:"synopsis"`
+	ImagesExposed *int64           `json:"images_exposed"`
+}
+
+type GetClusterCvesResponse []GetClusterCvesSelect
+
+// GetClusterCves represents /clusters/{cluster_id}/cves endpoint controller.
+//
+// @id GetClusterCves
+// @summary List of CVEs affecting/nonaffecting single cluster
+// @security RhIdentity || BasicAuth
+// @Tags clusters
+// @description Endpoint returning CVEs affecting/nonaffecting the given single cluster
+// @accept */*
+// @produce json
+// @Param cluster_id      path  string   true  "Cluster ID"
+// @Param sort            query []string false "column for sort"	  collectionFormat(multi) collectionFormat(csv)
+// @Param search          query string   false "cve name/desc search" example(CVE-2021-)
+// @Param limit           query int	   	 false "limit per page"		  example(10)
+// @Param offset          query int	   	 false "page offset"		  example(10)
```

Comment:
there is still two lines with same issue, probably it's because of tabs :D

Reply:
should be better now